### DISCUSSION TITLE
"Inaccessible or not found" error

### DISCUSSION THREAD

**Discussion by Bowlglet** [master-hax/pixel-backup-gang#21] [Q&A]
Hi,

I'm a complete noob here, but I keep getting this error whenever execute the commands from the instructions.  Mounting my drive does not seem to pop up anything like you mentioned.  Take a look at the screenshot for details.  Here is what I have and what I've done:

Google Pixel Sailfish
LineageOS 21 (Android 14)
Magisk Installed with root permissions applied to terminx
External drive with ext4 format

External drive seems to pop up in my notifications tab and able to access, but thats about it.  No mounting to the internal storage or anything like that.  Not sure what next steps to take from here, I'm
![Screenshot_20241015-234538_Termux](https://github.com/user-attachments/assets/6636c128-3f1d-44b6-8916-f904e08713c2)
 completely lost 😅  Any help would be appreciated!

**Comment by master-hax** [comment]
Please note that this is untested on Android 14

It looks like termux doesn't have root permissions because the calls to /sbin/su failed with "permission denied"

but that's weird because `su` works & shows a `#` prompt

could you try running `which su`? as a workaround you can run the steps from the install script manually.

**Reply by Bowlglet** [reply]
![Screenshot_20241016-095625_Termux](https://github.com/user-attachments/assets/5456377e-7115-44a2-936a-d3206d9f4016)

"/debug_ramdisk/su" shows up but don't know what that means. tried to follow w the next command and still the same error.

Could it be possible that I'm on LineageOS and not stock?

**Reply by angadbrar89** [reply]
I had the same issue you just have to do this command 

cd pixel-backup-gang

then you can do the rest

./start_global_shell.sh

./remount_vfat.sh /mnt/media_rw/ (name of drive)

**Reply by master-hax** [reply]
you need to `cd ./pixel-backup-gang` to move into the directory where the scripts are. you can use `pwd` to see your current directory and `ls` to see what is in the current directory.

**Reply by master-hax** [reply]
i pushed https://github.com/master-hax/pixel-backup-gang/commit/749a1545e602633d7d13d112c39591bdebcfd63c to make this clear in the README:
![image](https://github.com/user-attachments/assets/8f76d4bb-5a2c-406f-95a3-9aaac0bd60fc)
